deploy/openshift/templates/multi/keycloak-template.yaml

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2012-2018 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: che
+  annotations:
+    description: Che
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      io.kompose.service: keycloak
+    name: keycloak
+  spec:
+    replicas: 1
+    selector:
+      io.kompose.service: keycloak
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          io.kompose.service: keycloak
+      spec:
+        containers:
+        - env:
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: POSTGRES_PORT_5432_TCP_ADDR
+            value: postgres
+          - name: POSTGRES_PORT_5432_TCP_PORT
+            value: "5432"
+          - name: POSTGRES_DATABASE
+            value: keycloak
+          - name: POSTGRES_USER
+            value: keycloak
+          - name: POSTGRES_PASSWORD
+            value: keycloak
+          - name: KEYCLOAK_USER
+            value: admin
+          - name: KEYCLOAK_PASSWORD
+            value: admin
+          - name: PROTOCOL
+            value: "${PROTOCOL}"
+          - name: ROUTING_SUFFIX
+            value: "${ROUTING_SUFFIX}"
+          - name: CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD
+            value: "${CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD}"
+          image: '${IMAGE_KEYCLOAK}:${CHE_VERSION}'
+          command: ["/scripts/kc_realm_user.sh"]
```

Comment:
While testing I got `stat /scripts/kc_realm_user.sh: no such file or directory`. Isn't the right script `/scripts/keycloak_config.sh`?

Reply:
You need to test with `-p KEYCLOAK_IMAGE=eivantsov/keycloak`

Reply:
Ok got it thanks. I was able to start keycloak successfully.